src/main.py

```diff
@@ -19,22 +19,12 @@
 intents.members = True
 
 client = commands.Bot(command_prefix="/", intents=intents)
-client.remove_command("help")
 
-
-@client.command(name="send-welcome-survey")
-async def send_welcome_survey_command(context):
-    await admin_commands.send_welcome_survey(client, context)
-
-
-@client.command(name="ping-unanswered-survey")
-async def ping_unanswered_survey_command(context):
-    await admin_commands.ping_users_with_unanswered_questions(client, context)
-
-
-@client.command(name="delete-finished-surveys-channels")
-async def delete_finished_surveys_channels(context):
-    await admin_commands.delete_finished_surveys_channels(client, context)
+icogs = [
```

Comment:
I would prefer `_cogs`.

Reply:
Or perhaps it could even be inlined. What do you think @xjahic?

Reply:
_cogs is better imho as well and it doesn't matter if it is inlined or not

Reply:
Renamed in 084b414618bbe8b64220c99123438bb866ad3869